assets/i18n/en.json

```diff
@@ -72,26 +72,26 @@
     "I18N_ADD_NEW_SYLLABUS_ITEMS": "New Syllabus Items",
     "I18N_ADD_SYLLABUS_DESCRIPTION_TEXT": "Add skills or stories to your syllabus to automatically send them to your learners.",
     "I18N_ADD_SYLLABUS_SEARCH_PLACEHOLDER": "Search eg. Story,  Physics, English",
-    "I18N_ANDROID_PAGE_AVAILABLE_FOR_DOWNLOAD_TEXT": "available for download.",
-    "I18N_ANDROID_PAGE_BETA_DESCRIPTION": "A beta version of the Android app is now available in <strong>English</strong>,<br><strong>Brazilian</strong>, and <strong>Portuguese</strong>. The app is free, and no<br>subscription is needed.",
-    "I18N_ANDROID_PAGE_CONSENT_CHECKBOX_LABEL": "You confirm that you are over the age of 18 or that you have consent and approval from your legal parent or guardian.",
+    "I18N_ANDROID_PAGE_AVAILABLE_FOR_DOWNLOAD_TEXT": "Start learning for free on Android today",
+    "I18N_ANDROID_PAGE_BETA_DESCRIPTION": "A beta version of Oppia's Android app is now free to download and use in <strong>English</strong> and <strong>Brazilian Portuguese</strong>.",
+    "I18N_ANDROID_PAGE_CONSENT_CHECKBOX_LABEL": "I confirm that I am over the age of 18, or that I have consent and approval from my legal parent or guardian.",
     "I18N_ANDROID_PAGE_EMAIL_FIELD_LABEL": "Email address",
     "I18N_ANDROID_PAGE_FEATURES_SECTION_HEADER": "Education<br>for everyone.",
```

Comment:
Here and elsewhere, please avoid using br in translation strings. These should be handled in the CSS and not in the strings themselves.

Reply:
Noted. Done.